releasenotes/notes/28742.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: release-notes/v2
+kind: feature
+
+area: security
+
+issue:
+    - https://github.com/istio/istio/issues/28742
+
+releaseNotes:
+- |
+  **Added** Configuring Envoy to fetch the Jwks if the JwksUri is a mesh cluster URL for mTLS and other benefits. This is disabled by default 
```

Comment:
could you also mention this is an experimental feature for advanced users only? thanks